src/PeriodEnd/PaymentsDue/DeployPaymentsDue/sql/dml/03 PeriodEnd.Populate.Reference.Commitments.dml.sql

```diff
@@ -74,4 +74,8 @@ GO
 CREATE INDEX [IDX_Commitments_AccountId] ON Reference.DasCommitments (AccountId, CommitmentId, VersionId)
 GO
 
-CREATE INDEX IX_DasCommitments_Uln ON Reference.DasCommitments (Uln)
\ No newline at end of file
+CREATE INDEX IX_DasCommitments_Uln ON Reference.DasCommitments (Uln)
+GO
+
+CREATE INDEX IX_DasCommitments_CommitmentId ON Reference.DasCommitments (CommitmentId, VersionId)
```

Comment:
what happens if this index already exists from the last run?

Reply:
think we need this at the top like the others:

IF EXISTS (SELECT * FROM sys.indexes i
JOIN sys.objects t ON i.object_id = t.object_id
WHERE t.name = 'DasCommitments'
AND i.name = 'IX_DasCommitments_CommitmentId')
BEGIN
	DROP INDEX IX_DasCommitments_CommitmentId ON Reference.DasCommitments
END
GO

Reply:
This may be a stupid question, but why are we dropping and then rebuilding indexes?